pkg/apiserver/apiserver.go

```diff
@@ -67,12 +70,16 @@ func (c CompletedConfig) New() (*ServiceCatalogAPIServer, error) {
 	// giving it v1alpha1 version
 	apiGroupInfo.GroupMeta.GroupVersion = v1alpha1.SchemeGroupVersion
 
-	// TODO make storage work
-	//v1alpha1storage := map[string]rest.Storage{}
-	//
+	v1Alpha1Storage := map[string]rest.Storage{
+		"Broker":   broker.NewStorage(),
+		"Instance": instance.NewStorage(),
+		"Binding":  binding.NewStorage(),
+	}
+
+	// TODO: remove this?
 	//v1alpha1storage["servicecatalog"] = apiservice.NewREST(c.RESTOptionsGetter.NewFor(apiregistration.Resource("servicecatalog")))
```

Comment:
Yes. I've got a different version now, and I really do not understand how it's supposed to work, as what I have does not work.